src/main/java/com/redhat/darcy/salesforce/PicklistInputField.java

```diff
@@ -0,0 +1,111 @@
+/*
+ Copyright 2014 Red Hat, Inc. and/or its affiliates.
+
+ This file is part of darcy-salesforce.
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.redhat.darcy.salesforce;
+
+import static com.redhat.darcy.salesforce.RequiredInput.requiredInput;
+import static com.redhat.darcy.ui.By.chained;
+import static com.redhat.darcy.ui.By.nested;
+import static com.redhat.darcy.web.By.htmlTag;
+
+import com.redhat.darcy.ui.AbstractViewElement;
+import com.redhat.darcy.ui.Elements;
+import com.redhat.darcy.ui.annotations.Require;
+import com.redhat.darcy.ui.api.Locator;
+import com.redhat.darcy.ui.api.elements.Element;
+import com.redhat.darcy.ui.api.elements.Requireable;
+import com.redhat.darcy.ui.api.elements.Select;
+import com.redhat.darcy.ui.api.elements.SelectOption;
+import com.redhat.darcy.web.By;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * An HTML select element for a value that corresponds to a picklist field on 
+ * a Salesforce object.
+ */
+public class PicklistInputField extends AbstractViewElement 
+    implements Select<SelectOption>, Requireable {
+
+    @Require
+    private Element parent = super.parent;
+    
+    @Require
+    private Select<SelectOption> select = Elements.select(
```

Comment:
Yeah.  It gives a type safety warning if you don't static import Elements.  If you do static import, then the IDE seems to think you actually mean the public void select(locator) on line 97 and gives a Type mismatch error.